nautobot/docs/release-notes/version-1.5.md

```diff
@@ -30,6 +30,10 @@ As a result, the value of this setting now defaults to `False`, disabling databa
 !!! important
     Users with existing `nautobot_config.py` files generated from earlier versions of Nautobot will still have `CACHEOPS_ENABLED = True` unless they modify or regenerate their configuration. If users no longer desire caching, please be sure to explicitly toggle the value of this setting to `False` and restart your Nautobot services.
 
+#### Redesigned List Filtering UI
```

Comment:
Are regular `FilterForm` classes now completely obsolete, or do they still serve a purpose? If obsolete,

1) Are we planning to remove them from the code base in 1.5 or does that need to be a 2.0 item?
2) There's various developer documentation that needs to be revised, for example:

```
nautobot/docs/development/best-practices.md:| Table filter forms       | `NautobotFilterForm`       | `BootstrapMixin`           |
nautobot/docs/development/generic-views.md:    filterset_form = forms.ExampleModelFilterForm
nautobot/docs/development/style-guide.md:    * Similarly, for filter forms, `nautobot.extras.forms.NautobotFilterForm` combines `nautobot.utilities.forms.BootstrapMixin`, `nautobot.extras.forms.CustomFieldModelFilterFormMixin`, and `nautobot.extras.forms.RelationshipModelFilterFormMixin`, and should be used where appropriate.
nautobot/docs/models/extras/status.md:- `nautobot.extras.forms.StatusModelFilterFormMixin` should be used to add a non-required, multiple-choice `status` filter field to UI filter forms. This multiple-choice field allows for multiple status values to be selected for filtering objects in list views in the web UI.
nautobot/docs/models/extras/status.md:    In prior Nautobot versions these mixins were named `StatusFilterFormMixin` and `StatusBulkEditFormMixin`; the old names are still available as aliases but will be removed in a future major release.
nautobot/docs/models/extras/status.md:- Filter forms must inherit from `nautobot.extras.forms.StatusModelFilterFormMixin`
nautobot/docs/models/extras/status.md:    In prior Nautobot releases the latter two mixins were named `StatusBulkEditFormMixin` and `StatusFilterFormMixin` respectively; the old names are still available as aliases but will be removed in a future major release.
nautobot/docs/plugins/development.md:Plugins can extend any model-based `FilterSet` and `FilterForm` classes that are provided by the Nautobot core.
nautobot/docs/plugins/development.md:* The original `FilterForm` must follow the pattern: `f"{model.__name__}FilterForm"`, e.g. `TenantFilterForm`
nautobot/docs/plugins/development.md:* The `filterform_fields` attribute must be a valid dictionary of Django form fields
nautobot/docs/plugins/development.md:    A plugin is not required to define both `filterset_fields` and `filterform_fields`.
nautobot/docs/plugins/development.md:    filterset_form_class = YourPluginModelFilterForm
nautobot/docs/plugins/development.md:* `filter_form`: The FilterForm object for the associated model
nautobot/docs/plugins/development.md:    'filter_form': <ProviderFilterForm bound=True, valid=Unknown, fields=(region;site;location;q;asn;tag)>,
nautobot/docs/plugins/development.md:    filterset_form_class = YourPluginModelFilterForm
```

Reply:
I believe `FilterForm` would be obsolete @bryanculver ?

Reply:
Cause currently they don't serve any purpose

Reply:
I don't know where they will be displayed anymore. The intention of this is to expand the options of filtering as well as provide more room on the table.

I guess there's no reason we can't leave the `FilterForm`s in for 1.5 but 🤷

Thoughts @lampwins